src/core/src/op/constant.cpp

```diff
@@ -225,9 +235,14 @@ ov::op::v0::Constant::Constant(const Constant& other, const ov::Shape& new_shape
     m_data = other.m_data;
     update_identical_flags(other.m_all_elements_bitwise_identical_checked, other.m_all_elements_bitwise_identical);
     constructor_validate_and_infer_types();
+    m_femgr = ov::frontend::get_frontend_manager();
 }
 
-ov::op::v0::Constant::~Constant() = default;
+ov::op::v0::Constant::~Constant() {
```

Comment:
Looks like default destructor does the same things

Reply:
But we need keep m_data be released before frontend Manager.